source/server/admin/clusters_request.h

```diff
@@ -0,0 +1,128 @@
+#pragma once
+
+#include <cstdint>
+
+#include "envoy/buffer/buffer.h"
+#include "envoy/http/codes.h"
+#include "envoy/server/admin.h"
+#include "envoy/server/instance.h"
+
+#include "source/common/buffer/buffer_impl.h"
+#include "source/common/json/json_streamer.h"
+#include "source/server/admin/clusters_params.h"
+
+namespace Envoy {
+namespace Server {
+
+/**
+ * ClustersRequest captures context for a streaming request, implementing the
```

Comment:
I think you can omit "implementing the Admin::Request" interface as that's clear from the line below :)

maybe also say a `streaming /clusters request`?